libpius/path.py

```diff
@@ -99,11 +99,21 @@ def set_tmpdir(dir):
     if sys.platform == "win32":
         TMP = os.environ.get('TEMP')
         return os.path.join(TMP, dir)
```

Comment:
Handling Windows TEMP directory location, it appears that even if cmd is ran in administrator mode TEMP is still set to the current user.
(Possibly something to do with the env variables or how they are setup versus on other OSes). There should be no issue with finding the TEMP variable even as low as win XP has it. (It is set to the users %APPDATA%/local/temp/ folder be default)